scripts/plotting/meridional_mean.py

```diff
@@ -250,3 +294,18 @@ def _load_dataset(fils):
     else:
         sfil = str(fils[0])
         return xr.open_dataset(sfil)
+    #End if
+#End def
+
+def _get_climo_yrs(cam_ts_loc):
```

Comment:
If we save the start and end years as part of the ADF object in `adf_info.py` then this function and related code isn't needed.